src/pages/quiz/solving/QuestionsScreen.tsx

```diff
@@ -75,7 +75,14 @@ const QuestionsScreen: React.FC = () => {
 
     return (
         <View style={styles.container2}>
-            <TopBar/>
+            {/*TODO add here some title - likely description of what user does, f.e. quiz with question number*/}
+            <ModularTopBar
+                breadcrumbs={[
+                    {text: "<Quiz name>"},
+                    {text: "1/10"}
```

Comment:
should it be always 1/10?

Reply:
You have todo right above that:
`TODO add here some title - likely description of what user does, f.e. quiz with question number` xD
I just proposed how we could enhance the quiz solve view a little bit.
Ofc I can do that as part of this PR but I would like to extract that to another task/PR

Reply:
okey leave it as is for now